components/scoped-notification/index.jsx

```diff
@@ -0,0 +1,75 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+// Implements the [Scoped Notification design pattern](https://lightningdesignsystem.com/components/scoped-notifications/) in React.
+// Based on SLDS v2.4.5
+import IconSettings from '~/components/icon-settings';
+import Icon from '~/components/icon';
```

Comment:
The ~ import prefix should only be used in examples and not in components. I'm not sure what is documented anyway though. It's in the examples because a regular expression transform is run on them in the doc website.

Reply:
I have replaced those with
```
import IconSettings from '../icon-settings';
import Icon from '../icon';
```